package.json

```
@@ -4,6 +4,7 @@
   "private": true,
   "scripts": {
     "build": "next build",
+    "build:shadcn": "npm install -g shadcn@latest && npx shadcn@latest build",
```

I don't think installing shadcn globally is necessary. It adds unnecessary overhead since each workflow runs in isolation.

```suggestion
    "build:shadcn": "npx shadcn@latest build",
```